src/components/Chart.jsx

```diff
@@ -60,12 +60,14 @@ class Chart extends Component {
             topWidgets,
             showCountdown = false,
             chartContainerHeight,
+            containerWidth,
         } = this.props;
 
         const currentLang = lang || ((setting && setting.language) ? setting.language.key : 'en');
         t.setLanguage(currentLang);
         const currentPosition = `cq-chart-control-${(setting && setting.position && !isMobile) ? setting.position : 'bottom'}`;
         const currentMode = `${isMobile ? 'smartcharts-mobile' : ''}`;
+        const currentWidth =  !isMobile ? `smartcharts-${containerWidth}` : '';
```

Comment:
`currentWidth` is quite a confusing name. Can you come up with smth more descriptive? I'm also thinking it'll be better if you pass `smartcharts-${containerWidth}` as a prop instead of `containerWidth`

Reply:
@brucebinary As we do same for the theme, I think it's ok to pass only number